server/controllers/signupControllers/checkUser.js

```diff
@@ -0,0 +1,15 @@
+const { checkUserQuery } = require('../../database/queries');
+
+const checkUser = (userName, email) => checkUserQuery(userName, email)
+  .then((data) => data.rows)
+  .then((data) => {
+    if (data.length > 1) return 'userName and email are used by other users';
```

Comment:
this shouldn't be a possibility at all !!

Reply:
why?

Reply:
How could it be possible to have multiple rows with the same name and email!

Reply:
I think you miss understood the purpose, the possibility here is that the email is used for example by user1 and the username is used by another user for example user2